src/nlsolve_ac_powerflow.jl

```diff
@@ -153,5 +162,100 @@ function _nlsolve_powerflow(data::ACPowerFlowData; nlsolve_kwargs...)
     if !res.f_converged
         @error("The powerflow solver returned convergence = $(res.f_converged)")
     end
-    return res
+    return res.f_converged, res.zero
+end
+
+function _nlsolve_powerflow(pf::KLUACPowerFlow, data::ACPowerFlowData; nlsolve_kwargs...)
+    pf = PolarPowerFlow(data)
+    #J_function = PowerFlows.PolarPowerFlowJacobian(data, pf.x0)
+
+    maxIter = 30  # TODO
+    tol = 1e-6    # TODO
```

Comment:
It would be good to let these be configurable via the kwargs

Reply:
(and put the defaults into `const`s somewhere)

Reply:
Added the constants to definitions.jl